protos/sigstore_rekor.proto

```diff
@@ -128,3 +128,12 @@ message TransparencyLogEntry {
         // payload from other sources to verify the signature.
         bytes canonicalized_body = 7;
 }
+
+// The RekorBundle is the signed material used to produce the Signed Entry
+// Timestamp signature. See notes on the InclusionPromise above.
+message RekorBundle {
```

Comment:
Layout LGTM here, but I'm kind of murky on where this structure ends up being used (since it isn't referenced in the other messages). Is the idea that a tool like `rekor-cli` is expected to use it?

Reply:
I believe the idea is it could replace structure declarations like https://github.com/sigstore/sigstore-go/blob/main/pkg/tlog/entry.go#L260-L265, so that clients don't have to know the contents of the SET. They would still need to know the mapping between the TLE structure and this one though.

Reply:
Gotcha, makes sense!